README.md

```diff
@@ -1,5 +1,10 @@
-# Open Source Programs (Backend)
 ![Build Status](https://github.com/anitab-org/open-source-programs-backend/workflows/Tests%20Build/badge.svg)
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio)
+[![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
+![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=102)
```

Comment:
It's not visible in GitHub light mode, Can you change this?

Reply:
Can you provide a screenshot please? I can see the badges on my end.

![](https://i.gyazo.com/54a04116daa72058e5cbeaf2e0117449.png)

Reply:
![image](https://user-images.githubusercontent.com/56037184/108106469-fcf05d80-70b3-11eb-810a-84fcfa31befb.png)
@rmpsc Please see the attached image.

Reply:
@codesankalp Just committed a slightly different badge. Let me know if it resolves the problem.